docs/console/tutorials/create-extension.md

```diff
@@ -4,64 +4,81 @@ title: Getting started
 sidebar_label: Create your extension
 ---
 
-# Create your extension
+This tutorial is dedicated to the management of [extensions](/console/company-configuration/extensions.md),
+a feature that allows you to customize your company and/or your projects inside the Mia-Platform Console by including customized pages
+inside your Company page, the Project overview page or the Project Runtime page.
 
-Following this tutorial you will learn how to add the Backoffice as an extension within a Project, in order to consult it directly from the Console.
+And to do so, you will be using to create using only [miactl](/cli/miactl/10_overview.md), the command line interface application to communicate with the Mia-Platform Console APIs.
+
+## What you will build
+
+You are going to customize the sidebar of a Mia-Platform project by adding a new menu and a new link that, when clicked, will show an integrated Composer page.
+<!-- TODO: Should the Integration Connector Agent be part of the official Docs? Why can't I find it? -->
+This page will include a table with data extracted from an API coming from another project that uses the [Integration Connector Agent](https://github.com/mia-platform/integration-connector-agent/blob/main/docs/10_overview.md) to connect to a [Jira](https://www.atlassian.com/software/jira) Data Source.
```

Comment:
Should the Integration Connector Agent be part of the official Docs? Because I can't find it!